api/api/settings/prod.py

```diff
@@ -76,6 +77,18 @@
 
 WSGI_APPLICATION = "api.api.wsgi.application"
 
+# https://docs.djangoproject.com/en/5.0/ref/settings/#databases
+DATABASES = {
+    "default": {
+        "ENGINE": os.environ.get("DB_ENGINE", "django.db.backends.postgresql"),
+        "NAME": os.environ.get("DB_DATABASE", os.environ.get("RDS_DB_NAME")),
+        "USER": os.environ.get("DB_USER", os.environ.get("RDS_USERNAME")),
+        "PASSWORD": os.environ.get("DB_PASSWORD", os.environ.get("RDS_PASSWORD")),
+        "HOST": os.environ.get("DB_HOST", os.environ.get("RDS_HOSTNAME")),
+        "PORT": os.environ.get("DB_PORT", os.environ.get("RDS_PORT")),
+    }
+}
+
```

Comment:
I thought the algorithms api was stateless.
@Opeyem1a am I missing something?

Reply:
@keyvankhademi 

Intent is to store analytics data on calls to the algorithm API

Reply:
Currently storing request metadata (IP, time, URL path)